adr/README.md

```diff
@@ -0,0 +1,20 @@
+# Architecture Decision Records Folder
```

Comment:
Clarification question -- will a table of contents for ADRs be created (same was discussed during meeting that begat this PR)?  If so, how will the ToC be implemented -- single table for all ADRs, table for each directory of ADRs, ...?

Reply:
Will ToC in top level READMe be sufficient?

Reply:
Yes, I think it just needs to be on TOC at the top level.

Reply:
@lenny-intel If we have a single top-level ToC, will the references be somehow segmented by WG?  If not, would the overlap of WG-specific sequentially increasing reference numbers be confusing?  Will the ToC be in /adr proper?  Is the expectation that it be updated as part of the PR that adds an ADR?  Again, all clarifying questions in an attempt to minimize the confusion.

Reply:
Yes, I was thinking it would be segmented by sub-folders with the cross-cutting (no sub-folder) section first. Yes, I think it should be in /adr and yes, it should be updated as part of the PR to add a document.